ws281x_controller.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+from time import sleep
+import rpi_ws281x as neopixel
+
+class WS281XController():
+    """Controller for WS281X LED-strips connected to a Raspberry Pi.
+
+    Attributes:
+        led_count (int): Number of leds in the led strip.
+        led_pin (int): GPIO pin connected to the pixels (18 uses PWM!).
+        led_freq_hz (int): (Optional) LED signal frequency in hertz (usually 800khz).
+        led_dma (int): (Optional) DMA channel to use for generating signal (try 10).
+        led_invert (str): (Optional) True to invert the signal (when using NPN transistor level shift).
+        led_brightness (int): (Optional) LED signal frequency in hertz (usually 800khz).
+        led_channel (int): (Optional) set to '1' for GPIOs 13, 19, 41, 45 or 53..
```

Comment:
These are not attributes. The only attribute you have is `strip`. What you describe here are the arguments to the `__init__` method.